cmd/dev_server/start_server.go

```diff
@@ -28,17 +31,61 @@ func NewStartServerCmd(client dev_server.Client) *cobra.Command {
 
 	cmd.SetUsageTemplate(resourcescmd.SubcommandUsageTemplate())
 
+	cmd.Flags().String(cliflags.ProjectFlag, "", "The project key")
+	_ = viper.BindPFlag(cliflags.ProjectFlag, cmd.Flags().Lookup(cliflags.ProjectFlag))
+
+	cmd.Flags().String(SourceEnvironmentFlag, "", "environment to copy flag values from")
+	_ = viper.BindPFlag(SourceEnvironmentFlag, cmd.Flags().Lookup(SourceEnvironmentFlag))
+
+	cmd.Flags().String(ContextFlag, "", `Stringified JSON representation of your context object ex. {"user": { "email": "test@gmail.com", "username": "foo", "key": "bar"}}`)
```

Comment:
```suggestion
	cmd.Flags().String(ContextFlag, "", `Stringified JSON representation of your context object ex. {"kind": "multi", "user": { "email": "test@gmail.com", "username": "foo", "key": "bar"}}`)
```

I think this is invalid without the multi.